2018/002-update-channels.md

```diff
@@ -0,0 +1,123 @@
+ Title
+
+| Field  | Value      |
+|:-------|:-----------|
+| Status | Draft      |
+| Date   | 2018-07-31 |
+
+## Introduction
+
+### Problem description
+
+Automatic updates can be disruptive for a customer's application stack (ex.
+Kubernetes minor upgrade, default pod security policy introduction).
+
+Right now there is no visibility of what is inside an update and furthermore
+there is no possible selection of which updates are applied. An admin needs to
```

Comment:
This sounds like as we want to allow cherry picking of updates in the future. Do we really want that?
I thought the problem is more, that when we release intrusive updates, the customer cannot continue to apply critical bug fixes and security updates until he applied the intrusive updates.